_posts/2017-05-15-rust-at-two-years.md

```diff
@@ -0,0 +1,356 @@
+`git ls-tree HEAD text/ | wc` => there are 310 RFCs now
+310 - 255 =  55
+
+crates:
+`select count(*) from crates where date(created_at) >= '2016-05-15';`
+
+friends:
+https://github.com/rust-lang/rust-www/compare/bb7f26b...master#diff-ee76f5da00d603d5e34d053a3dc63395
+-->
+
+- 10,583 [commits] by 655 contributors added to the core repository;
+- 55 [RFCs] merged;
+- 9 major releases and 2 patch releases shipped;
+- 4,303 new [crates] published;
+- 284 library stabilizations;
+- 10 languages [rust-lang.org] has been translated into;
+- 48 new companies [running Rust in production][friends];
+- 2 new teams (Docs and Style);
+- 24 occasions of adding people to teams, 5 retirings of people from teams;
+- 3 babies born to people on [the Rust teams];
+- 2 years of [stability delivered].
+
+On an average week this year, the Rust community merged 1 RFC and published 83
+new crates. Rust topped the "[most loved] language" for the second year in a
+row in the StackOverflow survey. Also new this year is [thanks.rust-lang.org],
+a site where you can browse contributors by release.
+
+[Rust's first birthday]: https://blog.rust-lang.org/2016/05/16/rust-at-one-year.html
+[survey]: https://blog.rust-lang.org/2017/05/03/survey.html
+[most loved]: https://insights.stackoverflow.com/survey/2017#technology-most-loved-dreaded-and-wanted-languages
+[commits]: https://github.com/rust-lang/rust/commits/master
+[RFCs]: https://github.com/rust-lang/rfcs
+[rust-lang.org]: https://www.rust-lang.org/
+[friends]: https://www.rust-lang.org/en-US/friends.html
+[stability delivered]: http://blog.rust-lang.org/2014/10/30/Stability.html
+[thanks.rust-lang.org]: https://thanks.rust-lang.org/
+[the Rust teams]: https://www.rust-lang.org/en-US/team.html
+[crates]: https://crates.io/
+
+## Rust in production
+
+In addition to the 48 new Rust friends, we now have a [Rust jobs website]! More
+and more companies are choosing Rust to solve problems involving performance,
+scaling, and safety. Let's check in on a few of them.
+
+[Rust jobs website]: http://rustjobs.rs/
+
+[Dropbox] is using Rust in multiple high-impact projects to manage exabytes of
+data where correctness and efficiency is critical. Rust is currently shipping
+in the desktop client on Windows running on hundreds of millions of machines.
+Jamie Turner recently [spoke at the SF Rust Meetup] about the details on how
+Rust helps Dropbox use less RAM and get more throughput with less CPU.
+
+[Dropbox]: https://www.dropbox.com/
+[spoke at the SF Rust Meetup]: https://air.mozilla.org/rust-meetup-may-2017/
+
+Mozilla, Rust's main sponsor, has accelerated their use of Rust in production.
+Not only did [Servo start shipping nightly builds], [Firefox 48] marked the
+first Firefox release that included Rust code as part of the [Oxidation]
+project. [Project Quantum], announced in October 2016, is an effort to
+incrementally adopt proven parts of Servo into Firefox's rendering engine,
+Gecko.
```

Comment:
We could link to Potch's writeup as one possibility: https://hacks.mozilla.org/2017/05/quantum-up-close-what-is-a-browser-engine/

IMHO, I like that we just say we're incrementally adopting parts of Servo, and we don't necessarily call out specific ones.